modules/mobile/app/controllers/mobile/v0/letters_controller.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'common/exceptions/record_not_found'
+require 'evss/letters/download_service'
+require 'evss/letters/service'
+
+module Mobile
+  module V0
+    class LettersController < ApplicationController
+      before_action { authorize :evss, :access? }
+
+      def index
+        service_response = EVSS::Letters::Service.new(@current_user).get_letters
+        response_template = OpenStruct.new
+        response_template.id = @current_user.id
+        response_template.letters = service_response.letters
+        response_template.full_name = service_response.full_name
+        render json: Mobile::V0::LettersSerializer.new(response_template)
+      end
+
+      def beneficiary
+        service_response = EVSS::Letters::Service.new(@current_user).get_letter_beneficiary
+        response_template = OpenStruct.new
+        response_template.id = @current_user.id
+        response_template.benefit_information = service_response.benefit_information
+        response_template.military_service = service_response.military_service
+        render json: Mobile::V0::LettersBeneficiarySerializer.new(response_template)
+      end
+
+      def download
+        unless EVSS::Letters::Letter::LETTER_TYPES.include? params[:type]
+          Raven.tags_context(team: 'va-mobile-app') # tag sentry logs with team name
+          raise Common::Exceptions::ParameterMissing, 'letter_type', "#{params[:type]} is not a valid letter type"
+        end
+
+        response = EVSS::Letters::DownloadService.new(@current_user).download_letter(params[:type], request.body.string)
```

Comment:
Looks like the request body is passed through directly to the backend service. Does there need to be any protection on params passed thru?

Reply:
`{
"militaryService": true,
"serviceConnectedDisabilities": true,
"serviceConnectedEvaluation": true,
"nonServiceConnectedPension": true,
"monthlyAward": true,
"unemployable": true,
"specialMonthlyCompensation": true,
"adaptedHousing": true,
"chapter35Eligibility": true,
"deathResultOfDisability": true,
"survivorsAward": true
}`

This is the request body, would you expect any of these to need to be protected? I honestly have no idea if the body should be protected in this case or not.

Reply:
I was just reminded of the rails pattern to filter out unexpected query params before processing, and perhaps we should consider doing the same here. @kreek, is this a concern?